<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CANParamSetting</class>
 <widget class="QDialog" name="CANParamSetting">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>317</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CAN参数设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resonrces.qrc">
    <normaloff>:/images/app.ico</normaloff>:/images/app.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	color:#000;
	background-color: rgb(41,45,57);
}

QPushButton{
	height:25px;
	color:white;
	border:2px solid #147e3a;
	background-color:#019b48;
	border-radius:5px;
}

QPushButton:hover{
	background-color:#019b48;
	font-size:12px;
	font-weight:bold;
}

QLabel{
	color:white;
}

QGroupBox{
	border:2px solid rgb(41,45,57);
	color:black;
}

QComboBox{
	height:20px;
	border:1px solid rbg(41,45,57)
}

QMessageBox{
	background-color:rgb(41,45,57);
}

QMessageBox QPushButton:enabled{
	border-radius: 5px;
	width: 50px;
	height: 20px;
	background-color: #019b48;
	color: white;
	font-size: 12px;
}</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>291</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*下拉框样式*/
QComboBox{
	background-color: rgb(180, 200, 220);
	min-height: 20px;
	min-width: 60px;
	border: none;
	border-radius: 10px;
	color: rgb(20, 120, 50);
	font-family: &quot;Arial&quot;;
	font-size: 12px;
	padding-left:10px;
}

QComboBox::drop-down{
	width: 20px;
	border-left-width: 1px;
	border-left-color: rgb(120, 120, 120);
	border-left-style: solid;
}
QComboBox:hover{
	border:1px solid red;
}
QComboBox:!enabled {
    background-color: rgb(120, 120, 120);
	color: rgb(230, 240, 250);
}
QComboBox::down-arrow{
    image: url(://res/down arrow.png);
}
QComboBox::down-arrow:!enabled{
    image: none;
}
QComboBox::down-arrow:hover{
    image: url(://res/down arrow red.png);
}

/*按钮样式*/
QPushButton {
	width: 70px;
	height: 25px;
}
</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>驱动器连接端口</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="robotPortFrame">
      <property name="styleSheet">
       <string notr="true">/************/
QLabel{
	font-family: &quot;宋体&quot;;
	font-size: 14px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="labelAlignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="horizontalSpacing">
        <number>10</number>
       </property>
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="Label">
         <property name="text">
          <string>端口号</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="serialPortComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Label_2">
         <property name="text">
          <string>波特率</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="BaudRateComboBox">
         <property name="currentText">
          <string>115200</string>
         </property>
         <property name="currentIndex">
          <number>7</number>
         </property>
         <item>
          <property name="text">
           <string>1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>115200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256000</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Label_3">
         <property name="text">
          <string>奇偶校验</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="ParityComboBox">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Space</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mark</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Label_4">
         <property name="text">
          <string>数据位</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="DataBitsComboBox">
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="Label_5">
         <property name="text">
          <string>停止位</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="StopBitsComboBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connectSerialButton">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitBtn">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resonrces.qrc"/>
 </resources>
 <connections/>
</ui>
